ui/app/AppLayouts/Onboarding2/OnboardingLayout.qml

```diff
@@ -156,30 +177,25 @@ Page {
         passwordStrengthScoreFunction: root.onboardingStore.getPasswordStrengthScore
         isSeedPhraseValid: root.onboardingStore.validMnemonic
         validateConnectionString: root.onboardingStore.validateLocalPairingConnectionString
-        tryToSetPinFunction: root.onboardingStore.setPin
+        authorizeFunction: root.onboardingStore.authorize
         remainingAttempts: root.onboardingStore.keycardRemainingPinAttempts
 
         onKeycardPinCreated: (pin) => {
             d.keycardPin = pin
             root.onboardingStore.setPin(pin)
         }
 
-        onKeycardPinEntered: (pin) => {
-            d.keycardPin = pin
-            root.onboardingStore.setPin(pin)
-        }
-
-        onKeyPairTransferRequested: root.onboardingStore.startKeypairTransfer()
+        onLoadMnemonicRequested: d.loadMnemonic()
+        onAuthorizationRequested: d.authorize()
         onShareUsageDataRequested: (enabled) => root.shareUsageDataRequested(enabled)
         onReloadKeycardRequested: root.reloadKeycardRequested()
-        onMnemonicWasShown: root.onboardingStore.mnemonicWasShown()
-        onMnemonicRemovalRequested: root.onboardingStore.removeMnemonic()
```

Comment:
I understand we need to remove the mnemonic once the user has confirmed and verified it; no longer the case?